_posts/2019-02-06-snowplow-r112-baalbek-batch-pipeline-reliability-improvements/index.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: post
+title-short: Snowplow R112 batch pipeline reliability improvements
+title: "Snowplow R112 Baalbek batch pipeline reliability improvements"
+tags: [snowplow, batch]
+author: Ben
+category: Releases
+permalink: /blog/2019/02/06/snowplow-r112-baalbek-batch-pipeline-reliability-improvements/
+---
+
+[Snowplow 112 Baalbek][snowplow-release], named after
+[the city in Eastern Lebanon][baalbek], is a release focusing on reliability improvements for the
+batch pipeline.
+
+Please read on after the fold for:
+
+1. [EmrEtlRunner improvements](#eer)
+2. [Clojure Collector improvement](#cc)
+3. [Redshift (and Postgres) data model improvement](#redshift)
+4. [Upgrading](#upgrading)
+5. [Roadmap](#roadmap)
+6. [Help](#help)
+
+![baalbek][baalbek-img]
+<br>
+BlingBling10 at the English Wikipedia [CC BY-SA 3.0](http://creativecommons.org/licenses/by-sa/3.0/)
+
+<!--more-->
+
+<h2 id="eer">1. EmrEtlRunner improvements</h2>
+
+This release centers around EmrEtlRunner by adding new features whilst trying to make it more robust
+with respect to AWS services at the same time.
+
+<h3 id="persistent">1.1 Support for persistent EMR clusters</h3>
+
+EmrEtlRunner is now able to run steps on a long-running cluster saving up and down time through a
```

Comment:
Does up and down time mean the time it takes to spin up and tear down the resource?

Reply:
yes

Reply:
Would it be worth flagging that this opens up drip-feeding (i.e why I contributed that feature) and that it can yield sub 10 minute load times when used in combination with stream enrich mode?

Reply:
yup, I'm trying to be less verbose in blog post but that's definitely worth calling out.